### PR TITLE
feat: improve general clarity of CHALLENGE.md

### PR DESCRIPTION
**Pull request by i-oliva**
- Improve general copy
- Clarify that the tournament needs to be added on top of the list 
- Clarify that the loading state does not need to be handled when deleting (we have recently wrongly marked it as missing)

### CODE REVIEW

File: CHALLENGE.md [hunk 1]
```diff
@@ -52,71 +51,71 @@ The project includes:
...
-- Add a `CREATE TOURNAMENT` button.
+- Add a button labeled `CREATE TOURNAMENT`.
 
 ![create-tournament-prompt](.github/create-tournament-prompt.png)
 
-- Clicking on the `CREATE TOURNAMENT` button will open a browser prompt with a message `Tournament Name:`, an input field to enter the tournament name and the buttons `Cancel` to cancel and `OK` to confirm.
+- Pressing the `CREATE TOURNAMENT` button opens a browser prompt with the message `Tournament Name:`, an input field to enter the tournament name, and the buttons `Cancel` to cancel and `OK` to confirm.
 - The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
-- When confirming, the tournament will be created on the fake REST API and added on the UI without any loading indicators.
+- When `OK` is selected, the tournament is created on the simulated REST API and added to the top of the UI without any loading indicators.
```
Comment (alumedot) [inline CHALLENGE.md:106]: `to the top of the UI` sounds a bit strange..
Maybe `..at the start of the tournament list`, something like that

File: CHALLENGE.md [hunk 2]
```diff
@@ -138,9 +137,9 @@ Requirements:
...
 </details>
 
-## Finished?
+## Submission instructions
 
-- Create a `ZIP` file with your solution.
-- Include the .git folder to the `ZIP` file to make it easier for us to review your code.
-- Send it to the recruiter that sent you this challenge.
-- Please, don't include the `node_modules` folder in the `ZIP` file.
+- Compress your solution into a single `ZIP` file.
```
Comment (alumedot) [inline CHALLENGE.md:142]: The original line sounds easier in my opinion

**Review by alumedot — APPROVED** [review]
👍

File: CHALLENGE.md [hunk 3]
```diff
@@ -52,71 +51,71 @@ The project includes:
 
 ![edit-promp](.github/edit-prompt.png)
 
-- Clicking on the `EDIT` button will open a browser prompt with a message `New Tournament Name:`, an input field with the current tournament name as pre-filled value and the buttons `Cancel` to cancel and `OK` to confirm.
+- Pressing the `EDIT` button opens a browser prompt with the prompt message `New Tournament Name:`, an input field with the current tournament name as pre-filled value and buttons `Cancel` to cancel and `OK` to confirm.
 - The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
-- When confirming, the tournament name will be updated immediately using an optimistic update in the UI and a fetch call on the fake REST API.
+- When `OK` is selected, the tournament name updates immediately in the UI through an optimistic update and a fetch call to the fake REST API.
```
Comment (ejrcavenago) [inline CHALLENGE.md:56]: don't we want to mention `without any loading indicators`  also here? Same principle than the delete, still optimistic update

**Review by ejrcavenago — APPROVED** [review]
One comment about adding without loading indicators on edit as well, other than that is all good

**Pull request merged**